Examples/Tests/qed/breit_wheeler/analysis_core.py

```diff
@@ -183,6 +184,11 @@ def check_opt_depths(phot_data, ele_data, pos_data):
         assert( np.abs(scale - 1) < tol_red )
     print("  [OK] optical depth distributions are still exponential")
 
+def check_chi_at_creation(ele_data, pos_data, chi_theory):
+    assert(small_diff(ele_data["chi_at_creation"], chi_theory))
```

Comment:
Little wish, can you document this convention for the member here?
https://warpx.readthedocs.io/en/latest/developers/particles.html#particle-attributes

Reply:
Also, pls use camelCaseNames because we use currently the underscore sign in I/O to separate scalar from vector/tensor properties that have multiple components.

Reply:
These are good points, thanks. I've switched to camel case and added documentation in the page above (the formatting of the table is slightly off when I compile the doc but I see no simple way of improving this with my archaic rst knowledge :smiley: ).